lib/node_modules/@stdlib/array/complex64/docs/types/index.d.ts

```diff
@@ -195,6 +195,50 @@ type TernaryMapFcn<U> = ( this: U, value: Complex64, index: number, arr: Complex
 */
 type MapFcn<U> = NullaryMapFcn<U> | UnaryMapFcn<U> | BinaryMapFcn<U> | TernaryMapFcn<U>;
 
+/**
+* Reducer Function Invoked for each element in an array.
+*
+* @returns accumulated result
+*/
+type NullaryReducer<U> = ( this: U ) => ComplexLike;
```

Comment:
Rather than `ComplexLike`, this should be a generic. E.g.,

```
NullaryReducer<U, V> = ( this: U ) => V;
```

A reducer can reasonably return any accumulated value type. This applies here and below.